jobs/build/signed-compose/rpmdiff.py

```diff
@@ -33,18 +33,31 @@ def rpmdiffs_resolved(advisory):
     rpmdiffs = advisory.externalTests(test_type='rpmdiff')
     completed_diffs = []
     incomplete_diffs = []
+    failed_diffs = []
 
     print("Current RPM Diff status data:")
     pp(rpmdiffs)
 
     for diff in rpmdiffs:
-        if diff['attributes']['status'] in ['INFO', 'WAIVED', 'FAILED', 'NEEDS_INSPECTION']:
```

Comment:
You'll notice I'm the one that added `'FAILED', 'NEEDS_INSPECTION'`. What was happening _before_ was that if something came up with those statuses, it was considered incomplete, and so the job sat there and waited for them to change state until it timed out. These are complete, they're just failed.

Reply:
Moved those to the `failed diffs` section.